.circleci/config.yml

```diff
@@ -5,11 +5,19 @@ jobs:
     docker:
       - image: circleci/rust:1.41-stretch
     steps:
-      - run: rustup toolchain install nightly
+      - checkout
+      - run:
+          name: Setup Rust toolchain from ./rust-toolchain
+          command: echo "RUSTUP_TOOLCHAIN=$RUSTUP_TOOLCHAIN" && rustup show
       - run:
           name: Show version information
           command: rustc --version; cargo --version; rustup --version
-      - checkout
+      - run:
+          name: Add clippy component
+          command: rustup component add clippy
```

Comment:
👆 Install clippy component to currently active [nightly-2020-03-05  toolchain](https://github.com/CosmWasm/go-cosmwasm/blob/master/rust-toolchain). This must be executed with a nightly toolchain, since otherwise clippy cannot compile unstable code.